src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1326,7 +1326,22 @@ protected override void InvokeDefaultAction(string path)
                 invokeProcess.StartInfo.Arguments = path;
                 invokeProcess.Start();
 #elif CORECLR
-                throw new PlatformNotSupportedException();
+                if (Platform.IsNanoServer || Platform.IsIoT)
+                {
+                    throw new PlatformNotSupportedException();
+                }
+
+                invokeProcess.StartInfo.FileName = "cmd.exe";
+                // start is very picky: the "optional" TITLE as the first argument should always be included, otherwise it can silently fail.
+                // also, we need "" around the path in case that there is a space in the path.
+                invokeProcess.StartInfo.Arguments = string.Format(@"/c ""start /b """" ""{0}""""", path);
+                invokeProcess.Start();
+                invokeProcess.WaitForExit();
+                if (invokeProcess.ExitCode != 0)
+                {
+                    WriteError(new ErrorRecord(
```

Comment:
What about error in this case? `invokeProcess.Start();` (FullCLR) doesn't add anything to the ps errors.